extensions/functions_arithmetic_decimal.yaml

```diff
@@ -149,3 +149,19 @@ aggregate_functions:
         decomposable: MANY
         intermediate: "DECIMAL?<P, S>"
         return: "DECIMAL?<P, S>"
+  - name: "sum0"
+    description: >
+      Sum a set of values. The sum of zero elements yields zero.
+
+      Null values are ignored.
+    impls:
+      - args:
+          - name: x
+            value: "DECIMAL<P, S>"
+        options:
+          overflow:
+            values: [ SILENT, SATURATE, ERROR ]
+        nullability: DECLARED_OUTPUT
+        decomposable: MANY
+        intermediate: "DECIMAL?<38,S>"
```

Comment:
Based in the discussion in #465 the intermediate and return values should just be DECIMAL<38,S> since this function should not return null.

Reply:
Thanks for your review, I have submitted a new commit.

Reply:
> Based in the discussion in #465 the intermediate and return values should just be DECIMAL<38,S> since this function should not return null.

Thanks for your review, I have submitted a new commit according to the comment. @EpsilonPrime